src/Resources/config/services.yaml

```diff
@@ -0,0 +1,43 @@
+parameters:
+    landingi_toggle_redis_connection:
+        schema: 'tcp'
+        host: '127.0.0.1'
+        port: 6379
+
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    landingi_toggle.redis.client:
+        class: Predis\Client
+        arguments: [ '%landingi_toggle_redis_connection%' ]
+
+    Landingi\ToggleBundle\FeatureFlagsSource\RedisKeyFactory: ~
+
+    Landingi\ToggleBundle\FeatureFlagsSource\Cache\RedisCache:
+        public: false
+        arguments: ['@landingi_toggle.redis.client', '@Landingi\ToggleBundle\FeatureFlagsSource\RedisKeyFactory', '300']
+
+    Landingi\ToggleBundle\FeatureFlagsSource\DbSource:
+        public: false
+        arguments: ['@?']
```

Comment:
?

Reply:
Co to?

Reply:
Wywaliłem to jest pozostałość. Co do tego co to jest to ten znak definiuje, zeby Symfony automatycznie rozwiązało ten konkretny parametr, przydaje się jeśli chcesz np. zdefiniować ręcznie 3eci parametr ale dwa pierwsze niech symfony samo rozwiąże, wtedy używasz właśnie takiego syntaxa jak ten wyżej :D